packages/client/src/components/TeamMemberPhoto.jsx

```diff
@@ -0,0 +1,33 @@
+import { Avatar, Box } from '@mui/material';
+import { prependStrapiURL } from '../utils.jsx';
+
+export const TeamMemberPhoto = ({ Name, Photo, index }) => {
+  const colors = ['purple', 'yellow', 'blue'];
+  const selection = Math.floor(Math.random() * 3);
+  const placement = Math.floor(Math.random() * 3);
+  const firstColor = colors[selection];
+  const secondColor = colors[(selection + 1) % 3];
+  const thirdColor = colors[(selection + 2) % 3];
```

Comment:
Was this idea dropped or is this here for future use? Also, :star_struck: :star_struck: haha

Reply:
Oh I tried to add some color, in here but it didn't work. I will remove for now.